src/lib/update-alternatives/model/alternative.rb

```diff
@@ -0,0 +1,52 @@
+require "cheetah"
+
+module UpdateAlternatives
+  # Represents an alternative
+  class Alternative
+    attr_reader :name
+    attr_reader :status
+    attr_reader :value
+    attr_reader :choices
+
+    def initialize(name, status, value, choices)
+      @name = name
+      @status = status
+      @value = value
+      @choices = choices
+    end
+
+    def self.all_names
+      raw_data = Cheetah.run("update-alternatives", "--get-selections", stdout: :capture).lines
+      raw_data.map { |string| string.split.first }
+    end
+
+    def self.all
+      list = all_names
+      list.map { |name| query(name) }
+    end
+
+    def self.query(name)
```

Comment:
query is not good name. It reflect low level CLI call and not this object ability. I suggest something like `load(name` or `find(name)` (second is rails way)